data/fire_station_data.edn

```diff
@@ -1 +1 @@
7", :lat "51.47718863", :long "-0.199537206"} {:station "Hammersmith", :area_km2 "10.033813", :radius "1.7871379", :lat "51.49552858", :long "-0.222772918"} {:station "Chiswick", :area_km2 "8.554283", :radius "1.650125101", :lat "51.48983124", :long "-0.267505545"} {:station "Heston", :area_km2 "20.556845", :radius "2.558016222", :lat "51.47427845", :long "-0.341822225"} {:station "Feltham", :area_km2 "24.947539", :radius "2.817986568", :lat "51.4598878", :long "-0.412448576"} {:station "Hayes", :area_km2 "21.588468", :radius "2.621416181", :lat "51.49372423", :long "-0.43030331"} {:station "Heathrow", :area_km2 "15.733051", :radius "2.237852916", :lat "51.47944569", :long "-0.457522374"} {:station "Clapham", :area_km2 "6.852784", :radius "1.476925487", :lat "51.46418395", :long "-0.139567734"} {:station "Lambeth", :area_km2 "7.878804", :radius "1.583635439", :lat "51.4920181", :long "-0.119923212"} {:station "Brixton", :area_km2 "6.782396", :radius "1.469320829", :lat "51.46301509", :long "-0.108115448"} {:station "West Norwood", :area_km2 "14.36516", :radius "2.138357417", :lat "51.4251862", :long "-0.103924562"} {:station "Addington", :area_km2 "23.021024", :radius "2.706994557", :lat "51.35108058", :long "-0.022683268"} {:station "Battersea", :area_km2 "5.97549", :radius "1.379151022", :lat "51.46636556", :long "-0.167885516"} {:station "Woodside", :area_km2 "16.017031", :radius "2.257959104", :lat "51.38504334", :long "-0.060628235"} {:station "Purley", :area_km2 "30.612442", :radius "3.121576994", :lat "51.3310505", :long "-0.123644924"} {:station "Croydon", :area_km2 "16.172562", :radius "2.268895407", :lat "51.36810326", :long "-0.103640328"} {:station "Norbury", :area_km2 "14.281471", :radius "2.132119464", :lat "51.40604772", :long "-0.120015414"} {:station "Wandsworth", :area_km2 "16.991871", :radius "2.325657009", :lat "51.45577618", :long "-0.200077696"} {:station "Wimbledon", :area_km2 "15.432886", :radius "2.216402533", :lat "51.414799", :long "-0.196179825"} {:station "Tooting", :area_km2 "12.334748", :radius "1.981482332", :lat "51.43706304", :long "-0.16131744"} {:station "Mitcham", :area_km2 "14.077216", :radius "2.116817664", :lat "51.39664533", :long "-0.171420455"} {:station "Wallington", :area_km2 "19.842482", :radius "2.513176911", :lat "51.36162737", :long "-0.146844232"} {:station "Sutton", :area_km2 "20.019023", :radius "2.524332176", :lat "51.36828919", :long "-0.209746011"} {:station "Surbiton", :area_km2 "20.773406", :radius "2.571454938", :lat "51.39150971", :long "-0.296664384"} {:station "New Malden", :area_km2 "18.152957", :radius "2.403802337", :lat "51.39834884", :long "-0.243980855"} {:station "Kingston", :area_km2 "14.280161", :radius "2.132021675", :lat "51.42866546", :long "-0.305194454"} {:station "Richmond", :area_km2 "15.555469", :radius "2.225187535", :lat "51.46672988", :long "-0.283455013"} {:station "Twickenham", :area_km2 "27.974534", :radius "2.984052736", :lat "51.43444023", :long "-0.346214694"}]
```

Comment:
I thought it'd be easier when reading it to convert it into a dataset if it's a vector of maps.